playwright-tests/util/bos-loader.js

```diff
@@ -0,0 +1,43 @@
+export async function modifySocialNearGetRPCResponsesInsteadOfGettingWidgetsFromBOSLoader(
```

Comment:
Can we rename this function smaller? That is way too long.

Reply:
We can delete this file